src/main/java/com/innobridge/ethpay/service/TransactionService.java

```diff
@@ -94,5 +93,35 @@ public Transaction createTransaction(String senderId,
         return savedTransaction;
     }
 
+    public Transaction acceptTransaction(String transactionId) {
```

Comment:
pass in the receiver id, verify that the receiverid is the sender id in the transaction
perform withdraw from the sender account
perform deposit from the receiver account